addons/addon-base-raas/packages/base-raas-services/lib/helpers/__tests__/study-policy.test.js

```diff
@@ -35,10 +35,8 @@ describe('study-policy', () => {
           Action: [
             's3:GetObject',
             's3:GetObjectTagging',
-            's3:GetObjectTorrent',
```

Comment:
What's the reason for removing the Torrent permissions? Any risk that any customers were using it before?

Reply:
Will pointed out during doc review that this can be removed. I've tested that object access is not impacted after removing Torrent-related permissions even after upgrade install.